presto-main/src/main/java/com/facebook/presto/execution/DataDefinitionExecution.java

```diff
@@ -133,7 +132,7 @@ public void onFailure(Throwable throwable)
         catch (Throwable e) {
             fail(e);
             if (!(e instanceof RuntimeException)) {
```

Comment:
Why do we want to re-throw only checked exception? Seems to be just an artifact.

Reply:
The block doesn't throw any checked exceptions, so if it's not a `RuntimeException`, it must be an `Error`, which makes sense to re-throw. The original commit for that is fd87411ea2890cbaa8299ef32588a15c82e4304d.